tests/playbooks/tests_network_state.yml

```diff
@@ -0,0 +1,189 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+
+- name: Test configuring ethernet devices
+  hosts: all
+  vars:
+    type: veth
+    interface0: ethtest0
+    interface1: ethtest1
+
+
+  tasks:
+    - name: "set type={{ type }} and interface={{ interface0 }}"
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface0 }}"
+    - include_tasks: tasks/show_interfaces.yml
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+    - name: "set type={{ type }} and interface={{ interface1 }}"
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface1 }}"
+    - include_tasks: tasks/show_interfaces.yml
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+
+
+    - name: Configure the IP addresses
+      import_role:
+        name: linux-system-roles.network
+      vars:
+        network_state:
+          interfaces:
+            - name: ethtest0
+              type: ethernet
+              state: up
+              ipv4:
+                address:
+                  - ip: 192.168.122.250
+                    prefix-length: 24
+                dhcp: false
+              ipv6:
+                address:
+                  - ip: 2001:db8::1:1
+                    prefix-length: 64
+                autoconf: false
+                dhcp: false
```

Comment:
This is wrong, the `enabled` field is missing for IPv4 and IPv6 configuration. It should indicate `enabled: True` for both of them. Btw, Nmstate should raise a warning or error here and do not ignore it silently.